### PR TITLE
Adding Github templates

### PR DESCRIPTION
**Pull request by roigcarlo**
**Description of the PR**
This PR adds a couple of templates for PR and Issues to see if it helps to keep things clean.
As discussed with the @KratosMultiphysics/technical-committee it also suggest some fields as changelog etc...

**Changelog**
- Adding template for PR's
- Adding template for Issues

### CODE REVIEW

**Comment by roigcarlo** [comment]
Also notice that this is 100% optional but it will appear as a suggestion while making PR or Issues

**Comment by maceligueta** [comment]
Great, @roigcarlo ! I thought we could add a reminder for using **tags**. And a couple of questions that should be answered by the PR creator:
- It this PR an API-breaker? Yes/No (set Tag)
- Tests: does this PR need tests? Yes/No Are they included in the PR? Yes/No

**Comment by pooyan-dadvand** [comment]
@maceligueta a PR always needs tests!! The question would be if already has test coverage

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Scope of the bug**
+Where in Kratos is this bug happening. Ex
```
Comment (msandre) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Where in Kratos is this bug happening. E.g.,
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Compilation
+- KratosCore
+- FooApplication
+
+**Description the bug**
```
Comment (msandre) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
**Description of the bug**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+A clear and concise description of what the bug is.
```
Comment (msandre) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
A clear and concise description of the bug.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+**To Reproduce**
+Steps to reproduce the bug (if possible). Add here any custom example files as well in case they are relevant.
```
Comment (msandre) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Steps to reproduce the bug (if possible). Add any custom example files here if they are relevant.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment**
+Please specify your operating system, kratos branch and python version and any other library versions that you consider relevant.
```
Comment (msandre) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Please specify your operating system, kratos branch, python version and any other library versions that you consider relevant.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+Ex:
```
Comment (msandre) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
E.g.:
```

File: .github/ISSUE_TEMPLATE/pull-request.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+---
+name: Pull Request
+about: Create a PR to commit a new feature, bugfix or other change
+
+---
+
+**Description of the PR**
+A brief description of the PR. In case its a bugfix pleas add "Fix #1234" the issue or issues its solving
```
Comment (msandre) [inline .github/ISSUE_TEMPLATE/pull-request.md]:
```suggestion
A brief description of the PR. If it's a bugfix please add "Fixes #1234" to indicate the issue or issues it's solving.
```

File: .github/ISSUE_TEMPLATE/pull-request.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+**Changelog**
+Please, add here the change-log for the PR in form of a summarized list of changes.
```
Comment (msandre) [inline .github/ISSUE_TEMPLATE/pull-request.md]:
```suggestion
Please add here the change-log for the PR in the form of a summarized list of changes.
```

File: .github/ISSUE_TEMPLATE/pull-request.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+Ex:
```
Comment (msandre) [inline .github/ISSUE_TEMPLATE/pull-request.md]:
```suggestion
E.g.:
```

**Review by msandre — COMMENTED** [review]
Good choices for headings  :+1: I have some minor suggestions for the spelling and grammar.

**Comment by philbucher** [comment]
@roigcarlo is there a way to see the preview of this?

**Comment by roigcarlo** [comment]
@msandre Thx for the suggestions!
@philbucher no that i Know, but before merging I will put them in my private clone and link it here so you can play a little with them.

**Comment by philbucher** [comment]
> @philbucher no that i Know, but before merging I will put them in my private clone and link it here so you can play a little with them.

:+1: thx

**Comment by philbucher** [comment]
what is the status of this?

**Comment by roigcarlo** [comment]
Wasnt working on it, I have to change a couple of names but I will work on it thi week

**Comment by rubenzorrilla** [comment]
@roigcarlo what is the current status of this?

**Comment by roigcarlo** [comment]
I am gonna close the PR for now. I will leave the branch and reopen when its done.

**Comment by pooyan-dadvand** [comment]
Time to reopen it?

**Comment by philbucher** [comment]
still need the branch?

**Comment by roigcarlo** [comment]
Removed thx!

**Pull request merged**